packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -727,10 +734,15 @@ class _FloatingActionButtonTransitionState extends State<_FloatingActionButtonTr
 /// [bottomNavigationBar] is a [TabBar] and the body is a
 /// [TabBarView], you might be tempted to make each tab bar view a
 /// scaffold with a differently titled AppBar. It would be better to add a
-/// listener to the [TabController] that updates the AppBar:
+/// listener to the [TabController] that updates the AppBar.
+///
+/// ## Sample Code
+///
+/// Add a listener to the app's tab controller so that the [AppBar] title of the
+/// app's one and only scaffold is reset each time a new tab is selected.
 ///
 /// ```dart
-/// tabController = TabController(vsync this, length tabCount)..addListener(() {
+/// tabController = TabController(vsync tickerProvider, length tabCount)..addListener(() {
```

Comment:
Is this one missing `:` after `vsync` and `length`?

Reply:
Yes and oops. Thanks for pointing that out so quickly.